### ISSUE TITLE
Advanced Calculator App to Replace Simple Ones in Chromebooks (I'll handle the development, sit back and relax)

### ISSUE THREAD

**Issue by jljlspeedrunning** [Thornton-Middle-School/Project-Ideas#2]
I will make a calculator app, with all of the basic functionalities of a scientific calculator, plus graphing (including complex numbers), hex, bit, currency, and unit conversion. While the Chromebooks at school only carries a barely-functional calculating app, this app will be able to do much more.

**Comment by Pramad712** [comment]
Good idea but 1 thing: what's the problem with famous online calculators? For instance, Google's [Math Solver](https://www.google.com/search?si=ACC90nw1fyLplXVrTWLrPMRLsQrTy-TDlyAJO3N8Str86KoQJ5-T23BIl505guzzLVlpPS4q5q56gsGQLfiiaFDSjv9DVLwWplqhpgawvJT5KNtuS887MlNtQhDpGnN417jauXOPY302&sa=X&ved=2ahUKEwjnvfn7gs6IAxXYITQIHSRxB8cQi78NegQIBxAz), [Google Calculator](https://www.google.com/search?q=google+calculator&rlz=1C1ONGR_enUS981US981&oq=&gs_lcrp=EgZjaHJvbWUqBggBEEUYOzIGCAAQRRg5MgYIARBFGDsyDQgCEAAYgwEYsQMYgAQyBwgDEAAYgAQyDQgEEAAYgwEYsQMYgAQyBwgFEAAYgAQyBggGEEUYPDIGCAcQBRhA0gEINDkzNWowajeoAgCwAgA&sourceid=chrome&ie=UTF-8), [Unit Converter](https://www.google.com/search?q=pounds+to+kg&rlz=1C1ONGR_enUS981US981&oq=pounds+to+kg&gs_lcrp=EgZjaHJvbWUyDwgAEEUYORiDARixAxiABDIHCAEQABiABDIMCAIQABgUGIcCGIAEMgcIAxAAGIAEMgcIBBAAGIAEMgcIBRAAGIAEMgcIBhAAGIAEMgcIBxAAGIAEMgcICBAAGIAEMgcICRAAGIAE0gEINTk0OGowajeoAgCwAgA&sourceid=chrome&ie=UTF-8), or better yet, [Desmos](desmos.com)

**Comment by Pramad712** [comment]
(and also, I re-sent you the invitation to officially join this organization)
